### ISSUE TITLE
Default gh-pages branch

### ISSUE THREAD

**Issue by eah13** [skulpt/skulpt.github.io#1]
@bnmnetp thanks for setting this up!

In the settings of the repo you can make gh-pages the default branch.  (I don't have access to the settings). This will make sure that our new jekyll site will get deployed to skulpt.github.io and we can then delete master.  essentially gh-pages will be master for this repo and all pull requests will default to it, etc.

**Comment by bnmnetp** [comment]
Ok, I'll try to do that right away.

On another note, the use of Jekyll (heretical as it seems to use Ruby for a Python site) makes me suggest that maybe a monthly-ish blog style post would be cool.  We could highlight new developments, sites and other projects that use Skulpt and developing with skulpt style articles.

Someone would have to write them of course, but I think I could commit to a couple a year at least.

**Comment by bnmnetp** [comment]
OK, I made gh-pages the default.

**Comment by bnmnetp** [comment]
Now that I have done it, it seems to me that we don't need to use a different repository.  Can't we just create a gh-pages branch on the regular skulpt repository?  I think thats how a lot of projects do it.

**Comment by bnmnetp** [comment]
Yes, see:  https://help.github.com/articles/creating-project-pages-manually/

**Comment by eah13** [comment]
When I last looked into this, Project pages could not be used for custom domains, only custom subdomains.  It seems that may not be the case anymore.  Having the page be in a gh-pages branch in skulpt may muddy the dev waters a little (pull requests to the website will show up next to core dev for instance) but I'm finding out it'll be much much easier to get repo metadata into the site.

I'll do a pull request in skulpt and we can test it out there.  Thanks for finding this.

BTW we'll need an ANAME entry with our DNS provider:
https://help.github.com/articles/tips-for-configuring-an-a-record-with-your-dns-provider/

**Comment by eah13** [comment]
Pushed a gh-pages branch.  Site is live here: http://skulpt.github.io/skulpt/

Check the contributors page to see that nice API in action :)

OK presuming we can get the ANAME working with that site that's a better way to go.  I'll hold off on new commits until we figure that out.

Sorry I didn't have this 100% figured out from the beginning!
